### PR TITLE
Fix getControllerId() auto-generated id

### PR DESCRIPTION
**Pull request by machour**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | yes/no (not sure anymore, see #397)
| Tests pass?   | yes
| Fixed issues  | #397

### CODE REVIEW

**Comment by samdark** [comment]
1. Need a line for changelog.
2. Any idea why https://travis-ci.org/yiisoft/yii2-gii/jobs/491799386?

**Comment by samdark** [comment]
2.1 is going to be next release of gii.

**Comment by machour** [comment]
1. Changelog line added

2. There's something wrong with our PHPUnit constraint since b7becd5f43f610048b30c13d9ac29589b29b9661

If you look at https://travis-ci.org/yiisoft/yii2-gii/jobs/487945891 for example:
>   - Installing phpunit/phpunit (6.5.x-dev bac23fe): Downloading (100%)
> ...
> PHPUnit 7.5.1 by Sebastian Bergmann and contributors.

We are clearly not getting the expected PHPUnit version (composer.json asks for `< 7`)

3. @schmunk42 sorry I haven't had a chance to revise this PR according to your comment. Since this is going to be shipped in 2.1, is it okay with you as is ?

**Comment by schmunk42** [comment]
> We are clearly not getting the expected PHPUnit version (composer.json asks for `< 7`)

Is it using a pre-installed version? https://github.com/travis-ci/docs-travis-ci-com/issues/1016

> 3. @schmunk42 sorry I haven't had a chance to revise this PR according to your comment. Since this is going to be shipped in 2.1, is it okay with you as is ?

Is it OK, if I add a property `$strictInflector = true;`?

**Comment by machour** [comment]
> Is it OK, if I add a property `$strictInflector = true;`?

By all means, go ahead!

**Comment by machour** [comment]
Spot on for the Travis issue @schmunk42, thank you!

**Comment by schmunk42** [comment]
Everything is fine for me too.
Please go ahead with merging...

File: src/generators/crud/Generator.php [hunk 1]
```diff
@@ -43,6 +43,11 @@ class Generator extends \yii\gii\Generator
      * @since 2.0.5
      */
     public $enablePjax = false;
+    /**
+     * @var bool whether to use strict inflection for controller IDs
```
Comment (samdark) [inline src/generators/crud/Generator.php]: Worth explaining what "strict inflection" is.

Comment (machour) [reply src/generators/crud/Generator.php]: done

**Comment by samdark** [comment]
# #
Merged. Thank you!

**Pull request merged**